app/scripts/lib/app-start.js

```diff
@@ -107,7 +107,9 @@ function (
   User,
   FormPrefill,
   Notifications,
-  CloseButtonView
+  CloseButtonView,
+  config,
+  oAuthClient
```

Comment:
This changed from `OAuthClient` to `oAuthClient`. (not that its good or bad, just noting).